### ISSUE TITLE
Capacity depleted notification for Cron

### ISSUE THREAD

**Issue by dalnoki** [getsentry/sentry#71818]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Have more than one Cron monitors set up
2. Have an on-demand budget that well covers the charges for crons
3. You'll receive a Capacity depleted notification for Crons

### Expected Result

Since the on-demand budget fully covers the cron charges, users shouldn't receive this email

### Actual Result

Users currently receive the email which is confusing

### Product Area

Settings - Subscription

**Comment by brianthi** [comment]
Hi @dalnoki, can you share what the notification looks like? I assume you are referring to the notification that gets sent when included and reserved volumes have depleted. Since there's only 1 cron monitor included in the base plan it will always fire when you use that one cron

**Comment by JoshuaRVONQ** [comment]
@brianthi The conclusion that when you have more than 1 cron job, you will get those capacity depleted notifications. But we have reserved some money especially for cron monitoring.

So for detecting if the capacity is depleted for cron monitoring, the reserved money should be taken in consideration. As the price is about 70 cents per cron, we have reserved 125 USD. So the cron monitoring depletion message should be send when you are at about 90% of the reserved money. In this case around 110 USD.
So basically, detect depletion based on (reserved) money? In stead of hard/soft limit counts?

Hope this helps a bit to understand the issue. As we are getting those cron monitoring capacity depleted notifications

**Comment by brianthi** [comment]
Thanks, that helps @JoshuaRVONQ. We are currently working on a feature which will do just this, notify you at percentage thresholds of spend for each product category. These thresholds will be customizable, but will default to 80% and 100% consumed budget. We will reach out to you once this gets launched!

**Comment by JoshuaRVONQ** [comment]
Any updates. We are getting false email about cron monitoring stopping, which is not true.

**Comment by brianthi** [comment]
Hi @JoshuaRVONQ the feature I mentioned was launched, apologize for not updating this ticket. The way this is built right now, and with cron monitors only having 1 included cron on the subscription, you'll still unfortunately get the notification when that single monitor is used. However, you can customize the percentage threshold to get notified in /settings/billing/notifications/ for your on-demand usage. Hopefully that reduces some of the noise you're getting in notifications.
